### PR TITLE
Release 0.2.0

### PR DESCRIPTION
**Pull request by fredrikhgrelland**
This releases 0.2.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,9 @@ Portforwarding for nomad on port `4646` should bind to `127.0.0.1` and should al
 ### Starting a plain default box
 To get a running VM using the lastest release of this box run `vagrant box init fredrikhgrelland/hashistack` then `vagrant up`. The first command will add a file called `Vagrantfile` to your directory, and `vagrant up` will start a box based on the specifications of that file.
 
+### Starting a new project based on the hashistack
+This repository will release a new [template](template/README.md) into [fredrikhgrelland/vagrant-hashistack-template](https://github.com/fredrikhgrelland/vagrant-hashistack-template) on evry release.
+
```
Comment (m88614) [inline README.md:40]: Why is the title `Starting a new project based on the hashistack`?

File: README.md [hunk 2]
```diff
@@ -46,9 +49,6 @@ If you for any reason find yourself behind a transparent proxy you need to set t
 - Set the environment variables in your current session by running `export SSL_CERT_FILE=<path/to/ca-certificates-file>` and `export CURL_CA_BUNDLE=<path/to/ca-certificates-file>` in the terminal. Eg:`export SSL_CERT_FILE=/etc/ssl/certs/ca-certificates.crt CURL_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt`
 - Set the environment variables permanently by adding the export commands above to your `~/.bashrc` or equivalent.
 
-### Extending the base-box to fit your needs
-To see a full example of how to utilise this box and extend it, go to [template-repo](https://github.com/fredrikhgrelland/vagrant-hashistack-template).
-
```
Comment (m88614) [inline README.md:51]: Didn't see this before approving the first time, but why would you like to remove this?

File: README.md (hunk 1 above)
Comment (fredrikhgrelland) [reply README.md:40]: Do you have a better suggestion?

Comment (m88614) [reply README.md:40]: The text under doesn't have anything to do with starting a new project, does it?

File: README.md (hunk 2 above)
Comment (fredrikhgrelland) [reply README.md:51]: Moved it up and reworded it a bit. Thought that plain and templated should be together.

Comment (m88614) [reply README.md:51]: My bad, I didn't read `Starter template` section well enough.

File: README.md (hunk 1 above)
Comment (fredrikhgrelland) [reply README.md:40]: No it does not. This was a copy error.

**Pull request merged**